src/pages/meetingsPage.jsx

```diff
@@ -0,0 +1,113 @@
+import FusionLogo from '../assets/fusionLogo.png';
+import PlaceholderPic from '../assets/placeholderPic.png';
+
+export function MeetingsPage() {
+    return (
+        <div style={{
+            display: 'flex',
+            width: '1275px',
+            alignItems: 'flex-start',
+            alignContent: 'flex-start',
+            gap: '124px 231px',
+            flexWrap: 'wrap',
+            margin: 'auto',
```

Comment:
Might want to consider the architecture of your code here as though your os may have the light theme on, mine is in dark and leads to the website looking like the following photo, I tried fixing it on my end and got rid of the width and margin, however, this means that you may want to apply these properties to the other tags to ensure they look like what they look like in the figma 
![Screenshot 2024-07-06 211253](https://github.com/benson-fm/FUSION-Website/assets/72283466/7f781bbb-2755-4fe7-b3c3-bafe401d2ad4)